flixel/addons/plugin/screengrab/FlxScreenGrab.hx

```diff
@@ -155,28 +152,54 @@ class FlxScreenGrab extends FlxBasic
 		{
 			Filename = Filename + ".png";
 		}
+		return Filename;
+	}
+	
+	private static function save(Filename:String = ""):Void
+	{
+		if (screenshot.bitmapData == null)
+		{
+			return;
+		}
+		
+		Filename = fixFilename(Filename);
+		
+		var png:ByteArray;
+	#if flash
+		png = PNGEncoder.encode(screenshot.bitmapData);
+	#elseif openfl_legacy
```

Comment:
Oh I think I remember now. It's possible, with lime "hybrid" mode, to have the new lime backend but the old openfl, or vice versa, I forget. But it's possible for them to be mismatched so it is smart to articulate the one you actually mean.